pkg/controller/service/controller_test.go

```diff
@@ -345,23 +345,22 @@ func TestSyncLoadBalancerIfNeeded(t *testing.T) {
 				if !createCallFound {
 					t.Errorf("Got no create call for load balancer, expected one")
 				}
-				// TODO(@MrHohn): Clean up the awkward pattern here.
-				var balancer *fakecloud.Balancer
-				for k := range cloud.Balancers {
-					if balancer == nil {
-						b := cloud.Balancers[k]
-						balancer = &b
-					} else {
-						t.Errorf("Got load balancer %v, expected one to be created", cloud.Balancers)
+
+				var isFound bool
+				for _, balancer := range cloud.Balancers {
```

Comment:
Can we simply check on `len(cloud.Balancers)` to make sure only one LB is created?

Reply:
yeah~